com/bl/ttt/tictactoe.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+
+class Tictactoe:
+    """ Cell coordinates are:
+    0	1	2
+    3	4	5
+    6	7	8
+    """
+    def __init__(self):
+        self.cells = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+
+    def is_valid_move(self, player_id, move):
+        try:
+            if player_id == 'X' or player_id == 'O':
+                coordinate = int(move)
+                if 8 >= coordinate >= 0:
+                    if self.cells[coordinate] == ' ':
+                        valid_move_cell = coordinate
```

Comment:
I think this variable is redundant now it's not used anywhere.

Reply:
Yes. Note to self: review code for greyed out variables.

Reply:
Also `int(move.cell_chosen)`